app/src/main/kotlin/app/lawnchair/lawnicons/ui/component/ClickableIcon.kt

```diff
@@ -37,3 +39,18 @@ fun ClickableIcon(
         )
     }
 }
+
+@Composable
+fun ClickableIcon(
+    modifier: Modifier = Modifier,
+    onClick: () -> Unit,
+    painter: Painter,
+    tint: Color = MaterialTheme.colorScheme.onSurface,
```

Comment:
Why is the default value `onSurface` but is changed to `onBackground` on `IconPreview.kt`? Couldn't we just make it a constant value in the composable, since it's not changed anywhere else? (and we can easily do the "active icon" part if we do that)